python2/koans/about_lists.py

```diff
@@ -22,69 +22,69 @@ def test_list_literals(self):
         self.assertEqual([1], nums)
 
         nums[1:] = [2]
-        self.assertEqual([1, __], nums)
+        self.assertEqual([1, 2], nums)
 
         nums.append(333)
-        self.assertEqual([1, 2, __], nums)
+        self.assertEqual([1, 2, 333], nums)
 
     def test_accessing_list_elements(self):
         noms = ['peanut', 'butter', 'and', 'jelly']
 
-        self.assertEqual(__, noms[0])
-        self.assertEqual(__, noms[3])
-        self.assertEqual(__, noms[-1])
-        self.assertEqual(__, noms[-3])
+        self.assertEqual('peanut', noms[0])
+        self.assertEqual('jelly', noms[3])
+        self.assertEqual('jelly', noms[-1])
+        self.assertEqual('butter', noms[-3])
 
     def test_slicing_lists(self):
         noms = ['peanut', 'butter', 'and', 'jelly']
 
-        self.assertEqual(__, noms[0:1])
-        self.assertEqual(__, noms[0:2])
-        self.assertEqual(__, noms[2:2])
-        self.assertEqual(__, noms[2:20])
-        self.assertEqual(__, noms[4:0])
-        self.assertEqual(__, noms[4:100])
-        self.assertEqual(__, noms[5:0])
+        self.assertEqual(['peanut'], noms[0:1])
+        self.assertEqual(['peanut', 'butter'], noms[0:2])
+        self.assertEqual([], noms[2:2])
+        self.assertEqual(['and','jelly'], noms[2:20])
+        self.assertEqual([], noms[4:0])
+        self.assertEqual([], noms[4:100])
+        self.assertEqual([], noms[5:0])
```

Comment:
Q: What would the result be for `noms[5:0:-1]` be instead? Why?

Reply:
The result would be `['jellyy', 'and', 'butter' ]`, the reason is that the sliding is working like `[ )`, including the first term in the list and excluding the last, since there is no value in the position `5`, decreases till finds one in the position `3`, and since is not including the value in the position `0`, '`peanut` is leaved behind.